### PR TITLE
LPS-78093 We should wrap this method instead of moveNodeToTrash because the cache threadlocal is used in wikiPageLocalService.moveDependentToTrash

### PR DESCRIPTION
**Pull request by sergiogonzalez**

### CODE REVIEW

**Comment by sergiogonzalez** [comment]
cc/ @Preston-Crary

File: modules/apps/collaboration/wiki/wiki-service/src/main/java/com/liferay/wiki/service/impl/WikiNodeLocalServiceImpl.java [hunk 1]
```diff
@@ -573,10 +562,21 @@ public WikiNode updateStatus(
 	protected void moveDependentsToTrash(long nodeId, long trashEntryId)
 		throws PortalException {
 
-		List<WikiPage> pages = wikiPagePersistence.findByN_H(nodeId, true);
+		boolean clearCache = WikiCacheThreadLocal.isClearCache();
 
-		for (WikiPage page : pages) {
-			wikiPageLocalService.moveDependentToTrash(page, trashEntryId);
+		try {
+			WikiCacheThreadLocal.setClearCache(true);
```
Comment (Preston-Crary) [inline modules/apps/collaboration/wiki/wiki-service/src/main/java/com/liferay/wiki/service/impl/WikiNodeLocalServiceImpl.java]: Rebase error? False is now true?

**Comment by sergiogonzalez** [comment]
Yeah, copied from wrong file. Thanks @Preston-Crary

**Comment by sergiogonzalez** [comment]
https://github.com/brianchandotcom/liferay-portal/pull/55717

**Pull request closed without merging**